presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1757,4 +1759,17 @@ public FeaturesConfig setOffsetClauseEnabled(boolean offsetClauseEnabled)
         this.offsetClauseEnabled = offsetClauseEnabled;
         return this;
     }
+
+    public boolean isQueryOptimizationWithMaterializedViewEnabled()
+    {
+        return queryOptimizationWithMaterializedViewEnabled;
+    }
+
+    @Config("query-optimization-with-materialized-view-enabled")
+    @ConfigDescription("Enable query rewrite using materialized view. Please note that this only supports simple query formats for now.")
```

Comment:
Experimental: Enable query optimization using materialized view. It only supports simple query formats for now.